package.json

```diff
@@ -24,5 +24,8 @@
     "cypress:open": "PORT=4004 $(npm bin --prefix apps/site/assets)/cypress open",
     "test:cypress:run": "$(npm bin --prefix apps/site/assets)/start-server-and-test 'MIX_ENV=test PORT=4004 mix phx.server' 4004 cypress:run",
     "test:cypress:open": "$(npm bin --prefix apps/site/assets)/start-server-and-test 'MIX_ENV=test PORT=4004 mix phx.server' 4004 cypress:open"
+  },
+  "lint-staged": {
+    "**/*.{ex,exs,eex}": "mix format"
```

Comment:
`mix` only works at top level! I _think_ it would still work if I had included it in the other `package.json` file by writing it as `"../../../**/*.{ex,exs,eex}": "mix format"` but I felt like this both looked cleaner and was easier to understand.